Makefile

```
@@ -1,6 +1,14 @@
 build:
 	web-ext build -a dist
 
+build-chromium:
+	mkdir -p shiori-chromium
+	rsync -av css icons js view LICENSE shiori-chromium/
+	jq 'del(.background.scripts) | del(.browser_specific_settings) | .background += {service_worker: "js/service-worker.js"}' manifest.json > shiori-chromium/manifest.json
+	echo "importScripts('browser-polyfill.js', 'background-script.js');" > shiori-chromium/js/service-worker.js
+	sed -i 's#icons#/icons#g' shiori-chromium/js/background-script.js
+	web-ext build -s shiori-chromium -a dist
```

Maybe is better to build the chromium extension under a different directory?

```suggestion
	web-ext build -s shiori-chromium -a dist-chromium
```

---

Certainly, the zip filenames are the same, so using different directories would be better